modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -86,15 +86,15 @@ namespace
                     return gim.metadata(nh).get<NodeKind>().k == NodeKind::ISLAND;
                 });
 
-            const auto out_rois = cv::gimpl::getCompileArg<cv::GFluidOutputRois>(args);
+            const auto out_rois = cv::getCompileArg<cv::GFluidOutputRois>(args);
```

Comment:
looking at this, maybe this `getCompileArg<>` thing should be under `cv::gapi`

Reply:
Fixed